build.gradle

```diff
@@ -0,0 +1,203 @@
+plugins {
+  id 'checkstyle'
+  id 'jacoco'
+  id 'pmd'
+  id 'io.spring.dependency-management' version '1.0.6.RELEASE'
+  id 'org.springframework.boot' version '2.1.2.RELEASE'
+  id 'com.jfrog.bintray' version '1.8.4'
+  id 'org.owasp.dependencycheck' version '3.3.4'
+  id 'com.github.ben-manes.versions' version '0.17.0'
```

Comment:
there's newer versions of these plugins released, can you update to newest please.
there's a task that will show the versions for you
`./gradlew dependencyUpdates` I think otherwise run `./gradlew tasks` the correct one will be obvious